### ISSUE TITLE
Enhancement: Clarify maxbodylength messaging

### ISSUE THREAD

**Issue by mcandre** [mrtazz/checkmake#90]
There are many ways to calculate the length of some text. Bytes, characters, runes, words, lines...

I think the maxbodylength rule operates on lines, but the current warning string format does not appear to say this explicitly. Can we please add the word "line" just before "length" in the output?

**Comment by obnoxxx** [comment]
I fully agree that the violation message could/should be improved to clearly state that the limit is in number of **lines** 

I started to work on a patch but still facing issues to get the adjusted unit test to pass.
